### PR TITLE
Text added on hover.

### PR DESCRIPTION
**Pull request by rajdama**
Text added on hover successfully.

### CODE REVIEW

File: src/components/pets/pets.css [hunk 1]
```diff
@@ -56,3 +56,48 @@
...
+  position: relative;
+  width: 400px;
+}
+
+.image__img {
+  display: block;
+  width: 100%;
+}
+
+.image__overlay {
```
Comment (redxzeta) [inline src/components/pets/pets.css:70]: I think the overlay itself is a touch too dark. I'd lighten it a tad.

Can you add rounded corners too? u could use `border-radius`

File: src/components/pets/pets.css [hunk 2]
```diff
@@ -56,3 +56,48 @@
     5px 5px 15px 5px rgba(0, 0, 0, 0);
   box-shadow: 0px 10px 13px -7px #000000, 5px 5px 15px 5px rgba(0, 0, 0, 0);
 }
+
+.image {
+  position: relative;
+  width: 400px;
```
Comment (redxzeta) [inline src/components/pets/pets.css:62]: the width is kinda overlaps on mobile devices I think we can use 100%

File: src/components/pets/Pets.js [hunk 3]
```diff
@@ -56,8 +56,13 @@ export const AnimalType = ({ type, img, link }) => (
     md={6}
     className="pet__column d-flex flex-column align-items-center justify-content-end my-4"
   >
-    <Image src={img} alt={type} roundedCircle />
-    <Button as={Link} to={link} className="pet__button" color="primary">
+    <div className="image ">
+      <Image className="image__img" src={img} alt={type} roundedCircle />
+      <div className="image__overlay image__overlay--primary">
+        <div className="image__title">{type}</div>
```
Comment (redxzeta) [inline src/components/pets/Pets.js:62]: wondering if you change the className to `petType`

**Review by redxzeta — CHANGES_REQUESTED** [review]
Liking the changes so far :)

Just a couple more fixes :)

File: src/components/pets/pets.css (hunk 1 above)
Comment (rajdama) [reply src/components/pets/pets.css:70]: Border-radius of 15% added

File: src/components/pets/pets.css (hunk 2 above)
Comment (rajdama) [reply src/components/pets/pets.css]: Width changed to 100%

File: src/components/pets/Pets.js (hunk 3 above)
Comment (rajdama) [reply src/components/pets/Pets.js]: className changed from image to petType

**Review by redxzeta — APPROVED** [review]
nice changes :)

**Pull request merged**